### PR TITLE
Added PHPStan

### PR DESCRIPTION
**Pull request by Steveb-p**
| Question      | Answer
| ------------- | ---
| Tickets       | N/A
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ibexa/admin-ui/blob/main/LICENSE)

#### Checklist:
- [x] Implement tests
- [x] Coding standards (`$ composer fix-cs`)

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -10,13 +10,13 @@ on:
 jobs:
   cs_fix:
     name: Run code style check
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-22.04"
     strategy:
       matrix:
         php:
           - '8.0'
```
Comment (konradoboza) [inline .github/workflows/ci.yaml:17]:
```suggestion
          - '8.1'
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ jobs:
           - '8.1'
```
Comment (konradoboza) [inline .github/workflows/ci.yaml:47]:
```suggestion
          - '8.1'
          - '8.2'
```

File: src/lib/UserSetting/ValueDefinitionGroupRegistryEntry.php [hunk 3]
```diff
@@ -41,7 +41,7 @@ public function getDefinition(): ValueDefinitionGroupInterface
         return $this->definition;
     }
 
-    public function getIterator()
+    public function getIterator(): \Traversable
```
Comment (konradoboza) [inline src/lib/UserSetting/ValueDefinitionGroupRegistryEntry.php:44]: Import instead?

Comment (Steveb-p) [reply src/lib/UserSetting/ValueDefinitionGroupRegistryEntry.php]: Fixed in f3b4719a.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Steveb-p) [reply .github/workflows/ci.yaml]: Fixed in 498139b9.

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (Steveb-p) [reply .github/workflows/ci.yaml]: Fixed in 498139b9.

**Review by konradoboza — APPROVED** [review]

**Review by ciastektk — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Comment by adamwojs** [comment]
@Steveb-p Could you please merge up changed?

**Comment by Steveb-p** [comment]
> @Steveb-p Could you please merge up changed?

@adamwojs merged up in ca8da09e (4.5) & 0b82ed35 (main)

**Pull request merged**